src/code/z_scene.c

```diff
@@ -83,7 +232,11 @@ GLOBAL_ASM("./asm/nonmatching/z_scene/func_801306A4.asm")
 
 GLOBAL_ASM("./asm/nonmatching/z_scene/Scene_ProcessHeader.asm")
 
-GLOBAL_ASM("./asm/nonmatching/z_scene/Scene_CreateEntrance.asm")
+// func_80130768
+u32 Scene_CreateEntrance(u32 sceneIndex, u32 spawnIndex, u32 offset) {
+	return (((sceneIndex << 9) | (spawnIndex * 0x10)) | offset) & 0xffff;
```

Comment:
Instead of `spawnIndex * 0x10`, try `spawnIndex << 4` and see if it still matches, just to make it more consistent.